x/storage/keeper/cross_app_permission.go

```diff
@@ -146,16 +147,88 @@ func (app *PermissionApp) handleCreatePolicySynPackage(ctx sdk.Context, createPo
 	}
 
 	var policy permtypes.Policy
+	var grn types2.GRN
 	err = policy.Unmarshal(createPolicyPackage.Data)
 	if err != nil {
-		return sdk.ExecuteResult{
-			Payload: types.CreatePolicyAckPackage{
-				Status:    types.StatusFail,
-				Creator:   createPolicyPackage.Operator,
-				ExtraData: createPolicyPackage.ExtraData,
-			}.MustSerialize(),
-			Err: err,
+		var msgPutPolicy types.MsgPutPolicy
```

Comment:
I think define another type `MsgPutPolicy` is not friendly to the developer on BSC side, they have to distinguage `permtypes.Policy` and `types.MsgPutPolicy`, that is quite confusing. Please extend the `permtypes.Policy`

Reply:
accept,fixed